modules/gapi/test/common/gapi_video_tests_common.hpp

```diff
@@ -397,6 +397,36 @@ inline void testBackgroundSubtractorStreaming(cv::GStreamingCompiled& gapiBackSu
     EXPECT_FALSE(gapiBackSub.running());
 }
 
+inline void initKalmanParams(cv::gapi::KalmanParams& kp, int type, int dDim, int mDim, int cDim)
```

Comment:
Please add `const` keyword to all int parameters.

Reply:
Agree. Applied